src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -164,13 +164,9 @@ class OppgaveService(
         val oppgave = integrasjonerClient.hentOppgave(oppgaveId)
         if (skalFlyttesTilMappe(oppgave)) {
             val mappeId = mappeService.finnMappeIdForSøknadOgEnhet(søknadId, oppgave.tildeltEnhetsnr)
-            if (mappeId == null && oppgave.mappeId == null) {
-                log.info("Flytter ikke oppgave da den allerede er uplassert")
-            } else {
+            if (mappeId != null || oppgave.mappeId != null) {
```

Comment:
Vil vi faktisk bytte mappe for en oppgave som allerede har en mappe? Tror ikke jeg skjønner dét helt... 🤔

Reply:
Ja, hvis den skal flyttes ? Den blir oppdatert om den er lik også. Det er kanskje unødvendig ?

Reply:
Etter å ha sett igjennom koden "rundt" så er det vel kun når mappeId er ulik null vi trenger å sjekke på her. `oppgave.mappeId != null` er ulogisk - for da kommer vi aldri hit i utgangspunktet...

Reply:
Jepp. Sjekker bare om mappeId er null.